src/content/docs/en/guides/internationalization.mdx

```diff
@@ -0,0 +1,259 @@
+
+Use this function to retrieve a relative path for a locale. If the locale doesn't exist, Astro throws an error. 
+
+```astro title="src/pages/en/index.astro"
+---
+import {getRelativeLocaleUrl} from "astro:i18n";
+
+getRelativeLocaleUrl("fr", ""); // returns /fr
+getRelativeLocaleUrl("fr", "getting-started"); // returns /fr/getting-started
+getRelativeLocaleUrl("fr_CA", "getting-started", {
+  prependWith: "blog"
+}); // returns /blog/fr-ca/getting-started
+
+getRelativeLocaleUrl("fr_CA", "getting-started", {
+  prependWith: "blog",
+  normalizeLocale: false
+}); // returns /blog/fr_CA/getting-started
+
+---
+```
+
+### `getAbsoluteLocaleUrl()` 
+
+`getAbsoluteLocaleUrl(locale: string, path: string, options?: GetLocaleOptions): string`
+
+
+Use this function to retrieve an absolute path for a locale _if [`site`] has a value_. If [`site`] isn't, the function returns a relative URL. If the locale doesn't exist, Astro throws an error.
+
+
+```astro title="src/pages/en/index.astro"
+---
+import {getAbsoluteLocaleUrl} from "astro:i18n";
+
+// If `site` is set to be `https://example.com`
+
+getAbsoluteLocaleUrl("fr", ""); // returns https://example.com/fr
+getAbsoluteLocaleUrl("fr", "getting-started"); // returns https://example.com/fr/getting-started
+getAbsoluteLocaleUrl("fr_CA", "getting-started", {
+  prependWith: "blog"
+}); // returns https://example.com/blog/fr-ca/getting-started
+
+getAbsoluteLocaleUrl("fr_CA", "getting-started", {
+  prependWith: "blog",
+  normalizeLocale: false
+}); // returns https://example.com/blog/fr_CA/getting-started
+
+---
+```
+
+### `getRelativeLocaleUrlList()`
+
+It works like [`getRelativeLocaleUrl`](#getrelativelocaleurl), so it returns a list of relative paths for all the locales.
+
+
+`getRelativeLocaleUrlList(locale: string, options?: GetLocaleOptions): string[]`
+
+### `getAbsoluteLocaleUrlList()`
+
+`getAbsoluteLocaleUrlList(locale: string, options?: GetLocaleOptions): string[]`
+
+
+It works like [`getAbsoluteLocaleUrl`](#getabsolutelocaleurl), so it returns a list of relative paths for all the locales.
+
+
+## Routing strategy 
+
+The option `i18n.routingStrategy` allows to change the behaviour of the Astro routes
+
+### `'prefix-other-locales'`
+
+```js title="astro.config.mjs" ins={7}
+import { defineConfig } from "astro/config"
+export default defineConfig({
+  experimental: {
+    i18n: {
+      defaultLocale: "en",
+      locales: ["es", "en", "fr"],
+      routingStrategy: "prefix-other-locales"
+    }
+  }
+})
+```
+
+This is the **default** value. With this option all the routes *without* locales belong to the default locale (`i18n.defaultLocale`), and the rest of the locales must have their locale code (prefix) in the URL.
+
+- You want to have `example.com/content` for the default locale.
+- You want to have `example.com/es/content` for the other locales.
+- You want to have `example.com/` to redirect (302 status code) to `example.come/<i18n.defaultLocale>`.
```

Comment:
```suggestion

```

Isn't this in perfect contradiction with the first point here? If I'm missing something, there is an extra `to` before `redirect`.